tokenizers/src/tokenizer/pre_tokenizer.rs

```diff
@@ -51,7 +51,7 @@ impl From<(NormalizedString, Option<Vec<Token>>)> for Split {
 /// original string.
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct PreTokenizedString {
-    original: String,
+    pub original: String,
```

Comment:
I'm not seeing `.original` anywhere, why do you need this?

Also if it is for internal usage, `pub(crate)`

Reply:
good catch, I think I no longer need this!